src/main/java/uk/gov/hmcts/reform/pip/account/management/controllers/SystemAdminAccountController.java

```diff
@@ -39,18 +34,16 @@ public SystemAdminAccountController(SystemAdminAccountService systemAdminAccount
     /**
      * Create a system admin account for SSO user on the user table.
      *
-     * @param issuerId The ID of the user creating the accounts.
      * @param account The account to add.
      * @return The PiUser that's been added, or an ErroredPiUser if it failed to add.
      */
-    @ApiResponse(responseCode = OK_CODE, description = PI_USER)
-    @ApiResponse(responseCode = BAD_REQUEST_CODE, description = "{ErroredSystemAdminAccount}")
+    @ApiResponse(responseCode = "200", description = PI_USER)
+    @ApiResponse(responseCode = "400", description = "{ErroredSystemAdminAccount}")
     @ApiResponse(responseCode = "401", description = "Invalid access credential")
     @ApiResponse(responseCode = "403", description = "User has not been authorized")
     @PostMapping("/system-admin")
-    @PreAuthorize("@authorisationService.userCanCreateSystemAdmin(#issuerId)")
```

Comment:
I think we should keep the authorization for non-sso accounts

Reply:
This endpoint is only used to create SSO system account. The B2C system admin is created using the old system admin endpoint. In the previous ticket I done it like that to keep the 2 endpoints separate as B2C admin is going to be deprecated so it is easy to remove like that.

Reply:
I see. In which case, can we add some validation into here to ensure that only SSO users can be created by this new endpoint?